modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/Attachment/Attachment.tsx

```diff
@@ -3,181 +3,72 @@
-import ClayIcon from '@clayui/icon';
-import ClayLoadingIndicator from '@clayui/loading-indicator';
-import {
-	convertToFormData,
-	makeFetch,
-	useConfig,
-} from 'data-engine-js-components-web';
+import {ReactFieldBase as FieldBase} from 'dynamic-data-mapping-form-field-type';
 import {
 	FieldChangeEventHandler,
-	ReactFieldBase as FieldBase,
-} from 'dynamic-data-mapping-form-field-type';
-import {openSelectionModal, sub} from 'frontend-js-web';
-import React, {ChangeEventHandler, useRef, useState} from 'react';
-
-import './Attachment.scss';
-
-function validateFileExtension(
-	acceptedFileExtensions: string,
-	fileExtension: string
-) {
-	const isValidExtension = acceptedFileExtensions
-		.split(/\s*,\s*/)
-		.some(
-			(acceptedFileExtension) =>
-				acceptedFileExtension.toLowerCase() ===
-				fileExtension.toLowerCase()
-		);
-
-	if (!isValidExtension) {
-		return {
-			displayErrors: true,
-			errorMessage: sub(
-				Liferay.Language.get(
-					'please-enter-a-file-with-a-valid-extension-x'
-				),
-				acceptedFileExtensions
-			),
-			valid: false,
-		};
-	}
-}
-
-function validateFileSize(
-	fileSize: number,
-	maxFileSize: number,
-	overallMaximumUploadRequestSize: number
-) {
-	if (maxFileSize === 0 && fileSize > overallMaximumUploadRequestSize) {
-		return {
-			displayErrors: true,
-			errorMessage: sub(
-				Liferay.Language.get(
-					'file-size-is-larger-than-the-allowed-overall-maximum-upload-request-size-x'
-				),
-				`${overallMaximumUploadRequestSize / 1048576} MB`
-			),
-			valid: false,
-		};
-	}
-
-	if (
-		maxFileSize > 0 &&
-		fileSize > maxFileSize * overallMaximumUploadRequestSize
-	) {
-		return {
-			displayErrors: true,
-			errorMessage: sub(
-				Liferay.Language.get(
-					'please-enter-a-file-with-a-valid-file-size-no-larger-than-x'
-				),
-				`${maxFileSize} MB`
-			),
-			valid: false,
-		};
-	}
-}
-
-function File({attachment, loading, onDelete, readOnly}: IFileProps) {
-	if (loading) {
-		return (
-			<ClayLoadingIndicator className="lfr-objects__attachment-loading" />
-		);
-	}
-	else if (attachment) {
-		return (
-			<>
-				<div className="lfr-objects__attachment-title">
-					<ClayButton
-						displayType="unstyled"
-						onClick={() => {
-							window.open(attachment.contentURL, '_blank');
-						}}
-					>
-						{attachment.title}
-					</ClayButton>
-
-					<a
-						className="lfr-objects__attachment-download"
-						href={attachment.contentURL}
-					>
-						<ClayIcon symbol="download" />
-					</a>
-
-					{!readOnly && (
-						<>
-							<ClayButtonWithIcon
-								aria-label={Liferay.Language.get('delete')}
-								borderless
-								displayType="secondary"
-								monospaced
-								onClick={() => onDelete()}
-								symbol="times-circle-full"
-								title={Liferay.Language.get('delete')}
-							/>
-						</>
-					)}
-				</div>
-			</>
-		);
-	}
-
-	return null;
+	LocalizedValue,
+} from 'dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/types';
```

Comment:
I think we don't need to reference the full path because these interfaces are exported in [index.ts of this module](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/index.ts#L105).

Check examples [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/Attachment/Attachment.tsx#L15) and [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/js/MultiSelectPicklist/MultiSelectPicklist.tsx#L9).